bin/install_test_requirements.sh

```diff
@@ -36,6 +36,8 @@ pip install -e git+https://github.com/opendata-swiss/ckanext-harvester_dashboard
 pip install -r https://raw.githubusercontent.com/opendata-swiss/ckanext-harvester_dashboard/master/requirements.txt
 pip install -e git+https://github.com/opendata-swiss/ckanext-subscribe.git#egg=ckanext-subscribe
 pip install -r https://raw.githubusercontent.com/opendata-swiss/ckanext-subscribe/master/requirements.txt
+pip install -e git+https://github.com/opendata-swiss/ckanext-password-policy.git@1ed426aac36fb0cf7fb135b4501a05c515df4612#egg=ckanext-password-policy
+pip install -r https://raw.githubusercontent.com/opendata-swiss/ckanext-password-policy/1ed426aac36fb0cf7fb135b4501a05c515df4612/requirements.txt
```

Comment:
We can unpin the version of ckanext-password-policy once https://github.com/opendata-swiss/ckanext-password-policy/pull/7 is merged.